extensions/Gemini/python/pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+
+[project]
+name = "aiconfig_extension_gemini"
```

Comment:
It should have dashes instead of underscores, right? That's how the other extensions are, let's be consistent on names

Reply:
the pyproject.toml can not contain dashes, underscores are required here. This will break the package build system. The import statement also uses underscores
ex:
```python
import aiconfig_extension_gemini
```

Where as the pip install needs dashes
```
pip install aiconfig-extension-gemini
```

Reply:
Ah, understood. Thanks for the clarification